src/renderer/store/opPayloads.ts

```diff
@@ -15,8 +15,19 @@ export interface UndoChangeWordToSwampPayload {
   changedWord: Word;
 }
 
-export type DoPayload = DeleteEverySecondWordPayload | ChangeWordToSwampPayload;
+export interface DeleteWordsPayload {
+  startIndex: number;
+  endIndex: number;
+}
+
+export type UndoDeleteWordsPayload = DeleteWordsPayload;
+
+export type DoPayload =
+  | DeleteEverySecondWordPayload
```

Comment:
Can you please remove the dummy ops I wrote and their associated types now we have real ones 😅

Reply:
These all should be gone now, if you could please verify I got rid of everything that would be great!